base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {ebuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{sb} {sbuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{eb} {ebuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{ub} {ubuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebuc}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebuc}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebuc}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebuc}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebuc}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebuc}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebec}{ul} {ulec} {}
+\DeclareFontSeriesChangeRule {ebec}{el} {elec} {}
+\DeclareFontSeriesChangeRule {ebec}{l}  {lec}  {}
+\DeclareFontSeriesChangeRule {ebec}{sl} {slec} {}
+\DeclareFontSeriesChangeRule {ebec}{sb} {sbec} {bec}
+\DeclareFontSeriesChangeRule {ebec}{b}  {bec}  {}
+\DeclareFontSeriesChangeRule {ebec}{eb} {ebec} {bec}
+\DeclareFontSeriesChangeRule {ebec}{ub} {ubec} {bec}
+\DeclareFontSeriesChangeRule {ebec}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebec}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebec}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebec}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebec}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebec}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebec}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebec}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebc}{ul}  {ulc}  {}
+\DeclareFontSeriesChangeRule {ebc}{el}  {elc}  {}
+\DeclareFontSeriesChangeRule {ebc}{l}   {lc}   {}
+\DeclareFontSeriesChangeRule {ebc}{sl}  {slc}  {}
+\DeclareFontSeriesChangeRule {ebc}{sb}  {sbc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{b}   {bc}   {}
+\DeclareFontSeriesChangeRule {ebc}{eb}  {ebc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{ub}  {ubc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{uc}  {ebuc} {}
+\DeclareFontSeriesChangeRule {ebc}{ec}  {ebec} {}
+\DeclareFontSeriesChangeRule {ebc}{c}   {ebc}  {}
+\DeclareFontSeriesChangeRule {ebc}{sc}  {ebsc} {}
+\DeclareFontSeriesChangeRule {ebc}{sx}  {ebsx} {}
+\DeclareFontSeriesChangeRule {ebc}{x}   {ebx}  {}
+\DeclareFontSeriesChangeRule {ebc}{ex}  {ebex} {}
+\DeclareFontSeriesChangeRule {ebc}{ux}  {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsc}{ul} {ulsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{el} {elsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{l}  {lsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sl} {slsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sb} {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{b}  {bsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{eb} {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{ub} {ubsc} {bsc}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebsc}{ub} {ubsc} {ebsc}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.